OemPkg/OemConfigPolicyCreatorPei/OemConfigPolicyCreatorPei.c

```diff
@@ -18,10 +18,68 @@
 #include <Library/ConfigVariableListLib.h>
 #include <Library/ConfigKnobShimLib.h>
 #include <Library/SafeIntLib.h>
+#include <Library/ActiveProfileIndexSelectorLib.h>
 #include <ConfigStdStructDefs.h>
 
+// these externs are provided by PlatformConfigDataLib
 extern KNOB_DATA  gKnobData[];
 extern UINTN      gNumKnobs;
+// number of profile overrides (i.e. into gProfileData)
+// this does not count the generic profile, which is not
+// in gProfileData, but rather in gKnobData's defaults
+extern UINTN    gNumProfiles;
+extern PROFILE  gProfileData[];
+
+/**
+  Helper function to apply profile overrides. This function is only
+  called if the active profile index has been validated to be within
+  the acceptable range.
+  @param[in]  ActiveProfileIndex The validated index into gProfileData
+**/
+STATIC
+VOID
+ApplyProfileOverrides (

```

Comment:
Is there a plan for unit tests to verify overrides are applied the way we expect?

Reply:
mu_oem_sample currently does not have tests running. This would be a great place to add googletest support to enhance the repo and bring up usage of googletest :)